pysrc/bytewax/execution.py

```diff
@@ -30,7 +37,12 @@ def run(flow: Dataflow, inp: Iterable[Tuple[int, Any]]) -> List[Tuple[int, Any]]
 
         flow: Dataflow to run.
 
-        inp: Input data.
+        inp: Input data. If you are recovering a stateful dataflow,
+            your input should resume from the last finalized epoch.
```

Comment:
```suggestion
            your input should resume from the last finalized epoch specified.
```

Reply:
What is the `specified` getting at in this case? I imagine "last finalized epoch" is a fact about the dataflow, so only one exists at any given moment. And it's not part of the arguments here or something you need to coordinate between arguments.

Reply:
We reworded this to imply you have to make your input resume from the last finalized epoch.